src/app/reporting/Engine.h

```diff
@@ -135,6 +135,9 @@ class Engine
     CHIP_ERROR ScheduleBufferPressureEventDelivery(uint32_t aBytesWritten);
     void GetMinEventLogPosition(uint32_t & aMinLogPosition);
 
+    // Merges aAttributePath inside apAttributePathList if current path is overlapped with existing path in apAttributePathList
+    // Overlap means the path is superset or subset of another path
```

Comment:
This is where I would suggest moving that comment from the implementation above....

Reply:
updated, thanks